home.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Receitas da Hannah</title>
+        <link rel="stylesheet" href="home.css">
+    </head>
+    <body>
+        <h1>Receitas da Hannah</h1>
+        <h2>Join me on a flavor filled adventure!</h2>
+        <div class="boxes">
+            <a id="home" href="a">Home</a>
+            <a id="about" href="about.html">About</a>
+            <a id="brig" href="brigadeiro.hml">Brigadeiro</a>
+            <a id="tapioca" href="tapioca.html">Tapioca</a>
+            <a id="pudim" href="pudim.html">Pudim</a>
+        </div>
+        <h2>Recipes</h2>
+        <h3><a href="brigadeiro.html">Brigadeiro</a></h3>
+        <img src="https://milestone-1.netlify.app/images/caprese.jpeg">
```

Comment:
It would be great to see images of each of the dishes, you can just steal images of these dished from online!